connect/cli/plugins/commerce/utils.py

```diff
@@ -0,0 +1,340 @@
+    elif client.ns('pricing').streams.filter(id=stream_id).count() > 0:
+        return 'pricing'
+    return None
+
+
+def fill_general_information(ws, data, progress):
+    ws.title = 'General Information'
+    ws.column_dimensions['A'].width = 40
+    ws.column_dimensions['B'].width = 180
+    ws.merge_cells('A1:B1')
+    cell = ws['A1']
+    cell.fill = PatternFill('solid', start_color=Color('1565C0'))
+    cell.font = Font(sz=24, color=WHITE)
+    cell.alignment = Alignment(horizontal='center', vertical='center')
+    cell.value = 'Stream information'
+    count = len(data)
+    task = progress.add_task('Filling general information', total=count)
+    for line, key in enumerate(data, 2):
+        ws[f'A{line}'] = key
+        ws[f'B{line}'] = data[key]
+        progress.update(task, advance=1)
+
+
+def _fill_headers(ws, columns):
+    color = Color('d3d3d3')
+    fill = PatternFill('solid', color)
+    for n, value in enumerate(columns):
+        letter = string.ascii_uppercase[n]
+        cell = ws[f'{letter}1']
+        cell.fill = fill
+        cell.value = value
+
+
+def fill_columns(ws, columns, progress):
+    columns = list(columns)
+    ws.column_dimensions['A'].width = 22
+    ws.column_dimensions['B'].width = 20
+    ws.column_dimensions['C'].width = 40
+    _fill_headers(
+        ws,
+        (
+            'ID',
+            'Name',
+            'Description',
+            'Type',
+            'Position',
+            'Required',
+            'Output',
+        ),
+    )
+    alignment = Alignment(
+        horizontal='left',
+        vertical='top',
+    )
+    task = progress.add_task('Filling columns', total=len(columns))
+    for n, col in enumerate(columns, 2):
+        ws.cell(n, 1, value=col['id']).alignment = alignment
+        ws.cell(n, 2, value=col['name']).alignment = alignment
+        ws.cell(n, 3, value=col.get('description', '')).alignment = alignment
+        ws.cell(n, 4, value=col['type']).alignment = alignment
+        ws.cell(n, 5, value=col['position']).alignment = alignment
+        ws.cell(n, 6, value=col['required']).alignment = alignment
+        ws.cell(n, 7, value=col['output']).alignment = alignment
+        progress.update(task, advance=1)
+
+
+def fill_transformations(ws, transformations, progress):
+    transformations = list(transformations)
+    ws.column_dimensions['A'].width = 30
+    for column in ('B', 'C', 'D', 'E', 'F', 'G'):
+        ws.column_dimensions[column].width = 22
+    ws.column_dimensions['H'].width = 40
+    _fill_headers(
+        ws,
+        (
+            'ID',
+            'Function ID',
+            'Function Name',
+            'Description',
+            'Overview',
+            'Input columns',
+            'Output Columns',
+            'Position',
+            'Settings',
+        ),
+    )
+    alignment = Alignment(
+        horizontal='left',
+        vertical='top',
+    )
+    task = progress.add_task('Filling transformations', total=len(transformations))
```

Comment:
Before adding a progress, if the len of `transformations` is 0 you should return from this funciton